cmake/target/impl.cmake

```diff
@@ -37,7 +37,8 @@ function(add_module_target MODULE_NAME TARGET_NAME AC_INPUTS SOURCE_FILES AC_OUT
     add_custom_target("${TARGET_NAME}" 
       COMMAND ${CMAKE_COMMAND} -E chdir ${CMAKE_CURRENT_SOURCE_DIR}
       ${CMAKE_COMMAND} -E env PYTHONPATH=${PYTHON_AUTOCODER_DIR}/src:${PYTHON_AUTOCODER_DIR}/utils BUILD_ROOT="${FPRIME_BUILD_LOCATIONS_SEP}"
-      PYTHON_AUTOCODER_DIR=${PYTHON_AUTOCODER_DIR}
+      FPRIME_AC_CONSTANTS_FILE="${FPRIME_AC_CONSTANTS_FILE}"
+      PYTHON_AUTOCODER_DIR=${PYTHON_AUTOCODER_DIR} DICTIONARY_DIR=${DICTIONARY_DIR}
```

Comment:
`DICTIONARY_DIR` is unset and can be removed

Reply:
Good point.

Reply:
Given this is parallel to the codegen.py call, I'll take an action to clean this up after we are out of hotfix territory.